app/assets/stylesheets/welcome.scss.erb

```diff
@@ -60,8 +61,8 @@ html, body{
   border-style: solid #EED8AE;
```

Comment:
There's a way all the styles rules for the border can be written in one line. Same for background. Also delete the commented line 62 if its no longer in use